hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/__tests__/Overview.test.tsx

```diff
@@ -4,20 +4,20 @@
  * distributed with this work for additional information
  * regarding copyright ownership.  The ASF licenses this file
  * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
+ * 'License'); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
+ * distributed under the License is distributed on an 'AS IS' BASIS,
```

Comment:
Nit: Any reason of changing this to single quote? Is this due to the linting process?

Similar with the other files.

Reply:
Yes, the linter auto changed to single-quotes. Let me revert the license blocks.
Thanks for pointing this out to me

Reply:
Changed to double quotes in [6eaf4ab](https://github.com/apache/ozone/pull/6983/commits/6eaf4aba6eb1124384943f182ca1e6564cea3925)
Thanks